lib/handler/mongoHandler.js

```diff
@@ -152,7 +152,7 @@ module.exports = function (connectionString) {
       if (err) {
         return callback(err);
       }
-      const update = result.value;
+      const update = result;
```

Comment:
This change seems concerning. Was this required by the mongo update or something else?

Reply:
yes - it was from the mongo update